mkosi/__init__.py

```diff
@@ -2760,12 +2760,17 @@ def set_serial_terminal(args: CommandLineArguments, root: str, do_run_build_scri
     override_dir = os.path.join(root, "etc/systemd/system/serial-getty@ttyS0.service.d")
     os.makedirs(override_dir, mode=0o755, exist_ok=True)
 
+    columns = run(['tput', 'cols'], capture_output=True, text=True).stdout.strip()
+    lines = run(['tput', 'lines'], capture_output=True, text=True).stdout.strip()
+
```

Comment:
We don't need to shell out
```suggestion
    columns, lines = shutil.get_terminal_size(fallback=(80, 24))

```

Reply:
Nice!